src/pages/documentation/consent-for-edge-network/api-reference.md

```diff
@@ -65,4 +65,38 @@ iOS
 
 <Tabs query="platform=ios&api=update-consents"/>
 
+<InlineAlert variant="info" slots="text"/>
+
+The Consent extension uses Adobe Standard 2.0 when communicating with the Edge Network, when the updateConsents API is called.  Additionally, the property `metadata` is set to the time at which the API is called.
+
+Here is an example where updateConsents is called to set collect consent to y and bellow you can see the snippet of the request payload sent to the Edge Network.
+
+**Set and update Consent**
+
+```swift
+// Example in iOS (Swift),  updating users collect consent to 'yes'
+let collectConsent = ["collect": ["val": "y"]]
+let currentConsents = ["consents": collectConsent]
+Consent.update(with: currentConsents)
+```
+
+**Request payload sent to the Edge Network**
+
+```json
+    "consent": [
+        {
+        "standard": "Adobe",
+        "version": "2.0",
+        "value": {
+            "metadata": {
+            "time" : "2023-10-03T17:23:04.443Z"
+            },
+            "collect": {
+            "val": "y"
+            }
+          }
+        }
+     ]
+```
+
```

Comment:
Is there a reason we're not putting this within the tab?

Reply:
@calebk1 Thanks for providing me the updated InlineAlert script. Now we are able to group several texts and snippets together.
<img width="735" alt="updateAlert" src="https://github.com/AdobeDocs/aep-mobile-sdkdocs/assets/29806547/e9c24c94-0dc3-427d-85c6-d45c37728e7c">

Is there any naming reference we need to follow for slots?  I have
<InlineAlert variant="info" slots="text1, text2, text3, text4, text5"/>

Reply:
For the naming reference, the only rule that applies is that the text parts *must* start with text. So what you've done with text1, text2, text3, etc... is fine.

Reply:
Yup! Actually, this PR is the one that adds this capability (since I had to re-shadow the InlineAlert component), so once this is merged in, I'll make a PR that updates the info so it's more documented.